### DISCUSSION TITLE
recordfinder with filter scope?

### DISCUSSION THREAD

**Discussion by SledgehammerPL** [wintercms/winter#769] [Q&A] [answered]
Can I have recordfinder with filter? (image edited with pbrush to show what I want). And the result of selecting this filter would be removing line with valid_to 29 maj 2022 because it isn't valid today (15.11.2022). But when I change filter to 28.05.2022 it wouyd appear, but next one (30 maj 2022) would disappear...
(it is simple where ('valid_to','<',':filtered')->where('valid_from,'>':filtered') rule 

![image](https://user-images.githubusercontent.com/5888903/201953989-18bd3841-10a7-421f-b18c-618c3351cff9.png)

**Comment by LukeTowers** [comment, marked as answer]
You can set a scope on the recordfinder definition to have the effect of a "permanent" filter applied to the records displayed in that context, but it doesn't currently support defining it's own filter widget configuration. I wouldn't be opposed to a PR to add that support though, so if you need it feel free to make a PR :)
